examples/python/celsius_connectors/nisan_notes.md

```diff
@@ -0,0 +1,86 @@
+# example user controller
```

Comment:
Did you mean to delete this?

Reply:
@nisanharamati I assume it's safe to delete this?

Reply:
Yes, we can delete this flie